apps/src/studio/Item.js

```diff
@@ -119,6 +126,12 @@ Item.prototype.update = function () {
   // Draw the item's current location.
   Studio.drawDebugRect("itemCenter", this.x, this.y, 3, 3);
 
+  // In this stationary activity case, we don't need to do any of this
+  // update logic (facing the actor is handled every frame in display())
+  if (this.activity === 'watchActor') {
```

Comment:
We certainly could.  I'm tempted to hold off on that for a separate PR, so that we can get what's here in for recording this afternoon.